Makefile

```diff
@@ -0,0 +1,29 @@
+all: build check
+
+check: fmt vet lint ## Check your code
+
+revendor:
+	rm glide.lock
+	glide install --strip-vendor
+	glide-vc --use-lock-file --no-tests --only-code
+
+terraform:
```

Comment:
I'd rather this be a `.PHONY` `bin/terraform`, so the target matches the output, with `.PHONY` because we aren't declaring the upstream dependencies.

```Makefile
.PHONY: bin/terraform
bin/terraform:
  hack/get-terraform.sh
```

Pretty much everything in here is `.PHONY`.  I'm fine if you want to have a single entry at the end too:

```Makefile
.PHONY: \
  bin/openshift-install \
  bin/terraform \
  help \
  etc \
  etc
```

Reply:
For some stupid reason I had `.PHONY` everywhere and deleted it.